aptos-move/block-executor/src/executor.rs

```diff
@@ -509,6 +528,7 @@ where
         assert!(self.concurrency_level > 1, "Must use sequential execution");
 
         let versioned_cache = MVHashMap::new();
+        let shared_counter = AtomicU32::new(0);
```

Comment:
Is this the shared counter for generating ephemeral identifiers?

Reply:
yep! that's the plan